docs/system_overview/troubleshooting.md

```diff
@@ -2,15 +2,189 @@
 
 Some of the commonly encountered iRODS errors along with troubleshooting steps are discussed below.
 
-## The Server Log (rodsLog)
+## The Server Log
 
-The iRODS server log (rodsLog) is the best place to find a history of what has happened and any error codes and file paths that may explain unexpected behavior.  The rodsLog is found at `/var/lib/irods/log/rodsLog*`.  The rodsLog is rotated every few days, so make sure you're looking at the latest file for recent error messages.  The debugging level (below) affects how many messages are written to the rodsLog.
+As of iRODS 4.3.0, the server log file is managed by syslog. When unexpected behavior occurs, inspecting the log file is the first place that should be checked. By default, the log file is located at **/var/log/irods/irods.log**.
 
-## Debugging Levels
+iRODS provides default configuration files for **rsyslog** and **logrotate**. They can be found at the following locations:
```

Comment:
In a packaged installation, ...

Reply:
This is true for all cases, not just packaged installations.

This will need to be tweaked if that is not what we want.

Reply:
oh, fair enough.   resolving.